include/deal.II/base/quadrature_lib.h

```diff
@@ -439,6 +439,77 @@ class QSorted : public Quadrature<dim>
                   const std::pair<double, Point<dim> > &b);
 };
 
+/**
+ * Telles quadrature of arbitrary order.
+ *
+ * The coefficients of these quadrature rules are computed using
+ * a non linear variable change starting from a Gauss-Legendre
+ * quadrature formula.
+ * This is done using a cubic polynomial,
+ * $n = a x^3 + b x^2 + c x + d$
+ * in order to integrate
+ * a singular integral, with singularity at a given point x_0.
+ *
+ * We start from a Gauss Quadrature Formula with arbitrary
+ * function. Then we apply the cubic variable change.
+ * In his paper Telles applies the transformation on the reference
+ * cell [-1, 1] getting
+ * @f{align*}
+ * n(1) = 1
+ * n(-1) = -1
+ * dn/dx = 0 at x = x_0
+ * d2n/dx2 = 0 at x = x_0
+ * @f}
+ * We get
+ * @f{align*}
+ * a = 1/q,
+ * b = -3gamma_bar/q,
+ * c = 3gamma_bar/q,
```

Comment:
This won't look good in latex. You'll need to write it as \bar\gamma or similar.